aws/resource_aws_elasticache_security_group.go

```diff
@@ -17,6 +17,12 @@ func resourceAwsElasticacheSecurityGroup() *schema.Resource {
 		Create: resourceAwsElasticacheSecurityGroupCreate,
 		Read:   resourceAwsElasticacheSecurityGroupRead,
 		Delete: resourceAwsElasticacheSecurityGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				d.Set("name", d.Id())
```

Comment:
Instead of adding a custom importer, how do you feel about replacing `d.Get("name")` with `d.Id()`? It seems a bit cleaner as we may then use the simple passthrough importer here.